components/widgets/menu/menu_items/submenu_item.tsx

```diff
@@ -41,6 +41,9 @@ type Props = {
     ariaLabel?: string;
     root?: boolean;
     show?: boolean;
+    right?: boolean;
```

Comment:
Oh no...I already wrote this code in my sidebar changes...

Reply:
😅 I will rebase when you merge.

Reply:
Might be the other way around actually. Yours will likely get merged first, and yours also looks cleaner so I might just revert my change.